premium/assets/premium.html

```diff
@@ -33,7 +31,7 @@ <h2 class="card-title">Info</h2>
                     <section class="card">
                         <div class="card-body">
                             <ul>
-                                <li>Make a pledge on my <a href="https://patreon.com/yagpdb">Patreon</a>. Tiers above $3
+                                <li>Make a pledge on our <a href="https://patreon.com/yagpdb">Patreon</a>. Tiers above $3
                                     will grant you premium slots. It will take 2 minutes from you make a pledge to it
                                     being processed here.</li>
```

Comment:
10 minutes seems like a much more reasonable estimate for this.
```suggestion
                                    will grant you premium slots. It can take up to 10 minutes from you making a pledge to it
                                    being processed on our side. (If you do not see your premium slots after that time, please contact us through the support server.)</li>
```